dev/tasks/exec/tasks.js

```diff
@@ -13,29 +13,39 @@ const log = require( '../../utils/log' );
 const tools = require( '../../utils/tools' );
 const git = require( '../../utils/git' );
 
-module.exports = ( config ) => {
+/**
+ * Run task over ckeditor5 repositories.
```

Comment:
This API doc isn't formatted correctly. After each paragraph there should be a blank line. And code samples should be indented with two tabs (and no spaces).